File: dataset/Intermediate/Translation/Avatar/v1-Python-code/atcoder_ABC112_A.py
from sklearn.utils import shuffle
checkFirstCondition = 813
checkSubFirstCondition = 420
checkSecondCondition = 703
checkSubSecondCondition = 89
checkThirdCondition = 838
checkForthCondition = 924
shuffle([54, 92, 75])
from scipy.stats import ttest_ind
ttest_ind([11, 32, 69], [21, 12, 66])
age = int(input())
if checkFirstCondition & checkSubFirstCondition:
    if checkSecondCondition & checkSubSecondCondition:
        if checkThirdCondition & checkForthCondition:
            if age == 1:
                print('Hello World')
            else:
                (a, secondInput) = [int(input()) for iterator in range(2)]
                print(a + secondInput)

File: scripts/GAs/Experiment_Results_v1-Python-code/intermediate/SR/Translation/use_test/deepseek-coder-33b-instruct/Avatar/Python/Java-sanitized/atcoder_ABC112_A.java
import java.util.Scanner;

public class atcoder_ABC112_A {
    public static void main(String[] args) {
        int checkFirstCondition = 813;
        int checkSubFirstCondition = 420;
        int checkSecondCondition = 703;
        int checkSubSecondCondition = 89;
        int checkThirdCondition = 838;
        int checkForthCondition = 924;

        Scanner scanner = new Scanner(System.in);
        int age = scanner.nextInt();

        if ((checkFirstCondition & checkSubFirstCondition) != 0) {
            if ((checkSecondCondition & checkSubSecondCondition) != 0) {
                if ((checkThirdCondition & checkForthCondition) != 0) {
                    if (age == 1) {
                        System.out.println("Hello World");
                    } else {
                        int a = scanner.nextInt();
                        int secondInput = scanner.nextInt();
                        System.out.println(a + secondInput);
                    }
                }
            }
        }
    }
}